tests/kernel/common/testcase.yaml

```diff
@@ -2,3 +2,6 @@ tests:
   kernel.common:
     tags: kernel
     min_ram: 16
+  kernel.common.bootdelay:
+    extra_configs:
```

Comment:
I'm not sure 'common' is the right place to add this.
The test relies specifically on a non-default configuration which has to be enabled in an alternate config.
Because of this, we're going to be running all the other tests in the common suite a second time with this config turned on.
It might be better to split this out into its own standalone case.

Reply:
> It might be better to split this out into its own standalone case.

yep, on a second though, I agree with @andrewboie. Create a new test case for init.c and all things that can be done and configured during initialisation.

Reply:
also, common, is becoming crowded and does not fit on systems with limited RAM.